python-service/app/main.py

```
@@ -21,15 +21,24 @@
     allow_headers=["*"],
 )
 
-# OpenRouter経由でOpenAI互換APIを使用（idea-discussion/backendと同一のAPIキーを共有）
+# OpenRouter APIキーの取得（環境変数）
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)